src/v0/destinations/iterable/data/IterableTrackPurchaseConfig.json

```diff
@@ -11,7 +11,8 @@
   },
   {
     "destKey": "createdAt",
-    "sourceKeys": "originalTimestamp",
+    "sourceKeys": "createdAt",
+    "sourceFromGenericMap": true,
```

Comment:
same here - do we get createdAt all the time or should we have fallback to timestamp ?

Reply:
same here